feat(service): add seaweedfs template

Christopher2K opened the pull request:
## Changes

- added SeaweedFS template

## Issues

- Discussion: https://github.com/coollabsio/coolify/discussions/5368

> Note here that I added the default config for public read access. The documentation should mention how to remove that for people who don't need it.

Christopher2K committed:
feat: add seaweed docker yml

Christopher2K commented on templates/compose/seaweedfs.yaml:
```diff
@@ -0,0 +1,93 @@
+            ]
+          }
+    entrypoint: |
+      sh -c '\
+      sed "s/env:AWS_ACCESS_KEY_ID/$AWS_ACCESS_KEY_ID/g" /base-config.json > /base1-config.json; \
+      sed "s/env:AWS_SECRET_ACCESS_KEY/$AWS_SECRET_ACCESS_KEY/g" /base1-config.json > /config.json; \
+      weed server -dir=/data -master.port=9333 -s3 -s3.port=8333 -s3.config=/config.json\
```
Feels hacky but it does the job! Open to better suggestions.

ShadowArcanist commented on templates/compose/seaweedfs.yaml:
```diff
@@ -0,0 +1,93 @@
+      - "seaweed-admin-meta:/data"
+volumes:
+  seaweed-s3-data: null
+  seaweed-admin-meta: null
```
```suggestion
```
This is no needed, Coolify handles it

peaklabs-dev committed:
Merge branch 'next' into feat/add-seaweedfs-service

peaklabs-dev committed:
feat(service): improve seaweedfs

- improve healthchecks and remove start_period as it is not needed
- update seaweedfs to 4.05
- fix formatting, naming and order
- remove volumes section as coolify generates it

peaklabs-dev reviewed (approved):
Thanks for the PR.

If possible please also open the docs PR for this service.
